### PR TITLE
docs: version 1.0.0

### PR DESCRIPTION
**Pull request by mosheavni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -271,9 +272,9 @@ We use kubectl proxy and curl to reduce latency.
 
 ## ⚠️ Versioning
 
-> [!WARNING]
-> As we have not yet reached v1.0.0, we may have some breaking changes
-> in cases where it is deemed necessary.
+As we advance to `v1.0.0`, our primary goal is to maintain the stability of the
```
Comment (Ramilito) [inline README.md:275]: Nice!

**Review by Ramilito — APPROVED** [review]

**Comment by mosheavni** [comment]
added some info about informer

File: README.md (hunk 1 above)
Comment (mosheavni) [reply README.md:283]: thank ChatGPT for that lol

**Review by Ramilito — APPROVED** [review]

**Comment by Ramilito** [comment]
Let me know if it's ready!

**Comment by mosheavni** [comment]
it is :)

**Pull request merged**